packages/sdk/src/chains/Substrate/assetTransfer.ts

```diff
@@ -40,19 +41,18 @@ import { SubstrateFee, deposit, getBasicFee } from '.';
  *
  * <sign and send transfer>
  */
-export class SubstrateAssetTransfer {
+export class SubstrateAssetTransfer extends BaseAssetTransfer {
   private apiPromise!: ApiPromise;
 
-  public config!: Config;
-
   public async init(
     apiPromise: ApiPromise,
-    parachainId: SubstrateParachain,
     environment: Environment = Environment.LOCAL,
   ): Promise<void> {
     this.apiPromise = apiPromise;
+    const parachainId = apiPromise.consts.sygmaBridge.eip712ChainID as U256;
     this.config = new Config();
-    await this.config.init(parachainId.valueOf(), environment);
+    // This is probably not too safe and might overflow
```

Comment:
Overflow?

Reply:
Does it need to be a number in that case?

Reply:
could probably get away with a string to be honest